codecov_cli/commands/upload.py

```diff
@@ -181,29 +189,31 @@ def do_upload(
     ctx: click.Context,
     commit_sha: str,
     report_code: str,
+    branch: typing.Optional[str],
```

Comment:
Curious to know, did you manually order the args alphabetically, or does your linter do it for you automatically, or did you have to use a separate tool?

Reply:
manually

`:sort` in `vim`